docs/sources/dashboards/create-reports/index.md

```diff
@@ -107,11 +107,15 @@ You can include dynamic dashboards with panels or rows, set to repeat by a varia
 
 By default, reports use the saved time range of the dashboard. You can change the time range of the report by:
 
-- Saving a modified time range to the dashboard.
+- Saving a modified time range to the dashboard. Changing the dashboard time range without saving it doesn't change the time zone of the report.
 - Setting a time range via the **Time range** field in the report form. If specified, the custom time range overrides the time range from the report's dashboard.
 
 The page header of the report displays the time range for the dashboard's data queries. Dashboards set to use the browser's time zone use the time zone on the Grafana server.
 
+#### Report time zones
+
+To avoid confusing time zone behavior in your reports, we recommend against using the **Browser Time** time zone setting in the dashboard from which you're generating a report. Instead, set a specific time zone for the dashboard.
```

Comment:
```suggestion
To avoid confusing time zone behavior in your reports, we recommend against using the **Browser Time** time zone setting in the dashboard from which you're generating a report. Instead, set a specific time zone for the dashboard. That way reports always show the correct time zone regardless of what your default time zone is when you create a report.
```